lib/board.rb

```diff
@@ -0,0 +1,118 @@
+class Board
+  attr_reader :cells
+
+  def initialize
+    @cells = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ]
+
+    @moves = 0
+  end
+
+  def cell_available?(number)
+    false if number < 1 || number > 9
+    @cells.each do |row|
+      row.each { |cell| return true if cell == number }
+    end
+    false
+  end
```

Comment:
Your method is not working. I can select taken cells with no warning and I can also input a larger or smaller number with no warning. The only thing that it's doing it's not changing turns, but that is not enough, you should tell the player that the move was not valid.